Source/JavaScriptCore/jit/ExecutableAllocator.cpp

```diff
@@ -420,7 +420,8 @@ static ALWAYS_INLINE JITReservation initializeJITPageReservation()
         bool fastJITPermissionsIsSupported = false;
 #if OS(DARWIN) && CPU(ARM64)
 #if USE(INLINE_JIT_PERMISSIONS_API)
-        fastJITPermissionsIsSupported = (se_memory_inline_jit_restrict_with_witness_supported != nullptr)
+        // TODO: rdar://119669257
+        fastJITPermissionsIsSupported = /* (se_memory_inline_jit_restrict_with_witness_supported != nullptr) */ true
```

Comment:
Not sure about JSC, but I thought we don't normally land commented-out code like this. Maybe just `!!se_memory_inline_jit_restrict_with_witness_supported()`?